dev-tools/xtask/Cargo.toml

```diff
@@ -10,6 +10,7 @@ camino.workspace = true
 cargo_toml = "0.19"
 cargo_metadata = "0.18"
 clap.workspace = true
+macaddr.workspace = true
```

Comment:
We actually don't *need* this, but I'm using it to catch parseable errors earlier.